### ISSUE TITLE
Batch size of faster rcnn

### ISSUE THREAD

**Issue by thedarklord2** [rbgirshick/py-faster-rcnn#487]
Can someone please confirm if I understand this right?

In one training iteration you only feed the network with 1 or 2 images but with  128 or 256 GT labels.
So essentially one minibatch only has 1 or 2 images. I was skeptical because in other examples of neural networks I have seen a batch size typically 64. 

Any thoughts or suggestions would be greatly appreciated.
Thanks;

**Comment by makefile** [comment]
Yes,the original faster r-cnn implementation only support feeding 1 image at a time.If set iter_size:2 then total loss is averaged every 2 image,but still forward 1 original image at a time.
There is modifications supporting batch size larger than one. see #267 #51 .
